Changes to Q Developer content (#7396)
aws-amplify/docs: src/pages/[platform]/build-a-backend/q-developer/index.mdx

Q Developer in the IDE provides [inline code suggestions](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/inline-suggestions.html) in real time. As you write code, Amazon Q automatically generates suggestions based on your existing code and comments. When you start typing out single lines of code or comments, Amazon Q makes suggestions based on your current and previous inputs. Inline suggestions are automatically enabled when you download the Amazon Q extension.

## Setting up Q Developer

> Amazon Q is available as an extension in Visual Studio Code and a plugin in JetBrains. Amazon Q is also available in the AWS Toolkit for Visual Studio. To get started, please visit [Setup Amazon Q Developer](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/q-in-IDE-setup.html).

Reviewer: ```suggestion
Amazon Q is available as an extension in Visual Studio Code and a plugin in JetBrains. Amazon Q is also available in the AWS Toolkit for Visual Studio. To get started, please visit [Install Amazon Q Developer](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/q-in-IDE-setup.html).
```

> ## Use Q Developer - Inline code suggestions in your Gen 2 project

Reviewer: ```suggestion
## Use Q Developer - Inline code suggestions in your Amplify project
```

> Amplify generates two folders in our backend directory, `auth` and `data` which contain TypeScript CDK definitions for each of these resources. We’ll build out the schema for our API through the help of Amazon Q Developer - Inline code suggestion capabilities.

Reviewer: ```suggestion
Amplify generates two folders in your backend directory, `auth` and `data`, which contain TypeScript AWS CDK definitions for each of these resources. We’ll build out the schema for our API through the help of Amazon Q Developer's inline code suggestion capabilities.
```

// ...
```

> **Step 3:** Select the Inline code suggestion generated by Amazon Q developer. The Inline code suggestion feature assists you in defining the schema and hover over the output to select from other options.

Reviewer: ```suggestion
**Step 3:** Select the inline code suggestion generated by Amazon Q developer. The inline code suggestion feature assists you in defining the schema and hover over the output to select from other options.
```

<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
  <source src="/images/gen2/q-developer/Amplify-Gen2-Q-Developer.mp4" />
</video>

<Callout informational>

> **Note:** You can also trigger Inline code suggestion feature by invoking Amazon Q Developer manually using Option+C keyboard shortcut in VS Code. For more commands, please refer to the Commands tab in the Amazon Q extension.

Reviewer: ```suggestion
**Note:** You can also trigger inline code suggestion feature by invoking Amazon Q Developer manually using Option+C keyboard shortcut in VS Code. For more commands, please refer to the Commands tab in the Amazon Q extension.
```
